app/components/user-search.cjsx

```diff
@@ -16,12 +16,14 @@ module.exports = React.createClass
 
   searchUsers: (value) ->
     clearTimeout @queryTimeout
+    onSearch = @props.onSearch
 
     if value is ''
       Promise.resolve options: []
     else
       new Promise (resolve) =>
         @queryTimeout = delayBy @props.debounce, =>
+          onSearch() if onSearch
```

Comment:
This is related to Geordi because we are only trying to log geordi in certain instances of UserSearch. In this instance, the onSearch prop, which is a function, is passed in on line 59 of `inbox-form.cjsx`.